### DISCUSSION TITLE
Which ORM should be used in our application ?

### DISCUSSION THREAD

**Discussion by santoshshinde2012** [NeoSOFT-Technologies/rest-node-nestjs#302] [Q&A]
- [x] [MikroORM](https://github.com/mikro-orm/mikro-orm)
- [ ] [Prisma](https://github.com/prisma/prisma)
- [x] [TypeORM](https://github.com/typeorm/typeorm)
- [ ] No ORM

Please provide a reason for your suggestion .

**Comment by yashesh-neo** [comment]
Typeorm is more versatile and supports RDBMS well but not specially built for Typescript based backend, MikroORM has some good design patterns as well as better MongoDB support.
